pyccel/ast/internals.py

```diff
@@ -37,7 +37,7 @@ def is_elemental(self):
         """ Indicates whether the function should be
         called elementwise for an array argument
         """
-        return False
+        return True
```

Comment:
Please do not change the default behaviour. The property should be overriden in `NumpyImag` not changed here